app/javascript/gobierto_dashboards/modules/contracts_controller.js

```diff
@@ -1,15 +1,25 @@
 import Vue from "vue";
 import VueRouter from "vue-router";
+
+import { sum, mean, median, max } from 'd3-array';
+import { scaleThreshold } from 'd3-scale';
+const d3 = { scaleThreshold, sum, mean, median, max }
+
+import crossfilter from 'crossfilter2'
+
 import { getRemoteData } from '../webapp/lib/get_remote_data'
 import { EventBus } from '../webapp/mixins/event_bus'
 import { money } from 'lib/shared/modules/vue-filters'
-import { sum, mean, median, max } from 'd3-array';
 
-const d3 = { sum, mean, median, max }
+import { AmountDistributionBars } from "lib/visualizations/modules/amount_distribution_bars.js";
+import { GroupPctDistributionBars } from "lib/visualizations/modules/group_pct_distribution_bars.js";
```

Comment:
Do not access to the elements by a path. They should be accessible in that way:

```suggestion
import { AmountDistributionBars } from "lib/visualizations";
import { GroupPctDistributionBars } from "lib/visualizations";
```

To make that work you'll need to `export` the elements in their respective files, and `import`/`export` them in the index file

Do the same with the `money` function above.

Doing this we don't depend where the file is stored.

Reply:
Ahh, didn't know that. Fixed. Thanks!